content/doc/book/security/index.adoc

```diff
@@ -9,30 +9,75 @@ ifdef::backend-html5[]
-
-Many of the security options are enabled by default when passing the interactive setup wizard to ensure that Jenkins is secure.
-Others involve environment-specific setup and trade-offs and depend on specific use cases supported in individual Jenkins instances.
-
-This chapter will introduce the various security options available to Jenkins administrators and users, explaining the protections offered, and trade-offs to disabling some of them.
+Jenkins is used everywhere -- from workstations on corporate intranets to high-powered servers connected to the public internet.
+It is critically important to keep your Jenkins instance secure,both to protect your information and to avoid executing malicious code from your Jenkins instance.
+
+Your Jenkins environment is a fully-distributed build system.
+Each network connection is a potential point of entry.
+Remember that the code that runs your builds can be perverted to run anything!
+
+Jenkins and the jobs it runs must be able to do almost anything.
+This means that a malicious Pipeline could reconfigure the Jenkins instance, delete files, or launch various forms of mischief such as a DDoS attack or a bot.
+In addition to deliberate and direct attacks on your environment, a trusted user could visit an infected web site and accidentally introduce malicious code into the Jenkins instance.
+
+Jenkins includes configurable features to secure your Jenkins instance against the various security and threat profiles.
+The setup wizard enables many of the security options by default,
+to ensure that Jenkins is secure.
+Other security options involve environment-specific setup and trade-offs and depend on specific use cases supported for individual Jenkins instances.
+Configuration options allow you to enable, customize, or disable security features.
+
+[CAUTION]
+====
+Most Jenkins environments grow over time, requiring their trust models to evolve as the environment grows.
+You should schedule regular "check-ups" of your security settings to ensure that they are still appropriate for your instance.
+In particular, if you had to disable some of the default protections that Jenkins provides,
+perhaps because you were using a plugin that had not been updated to support that protection, you may be able to re-enable that protection to increase the security of your instance.
+====
 
 // TODO the following only makes sense on the web site, not the PDF. Can it be disabled there?
+// TODO the material below should be moved to other sections in this chapter.
+
+This chapter introduces the various security options available to Jenkins administrators and users, explaining the protections offered, and trade-offs to disabling some of them:
 
-== Basic Setup
+link:concepts-security[Background Concepts]::
+Discusses security principles that should guide all your decisions about security
+and explains how Jenkins executes a Pipeline so you understand where intrusions can occur if your security configuration is lax.
```

Comment:
I wonder whether we can further split up the entries in this "definition" list into separate subheadings, as the existing ones all discuss a specific feature of Jenkins and its security implications, each having either "this needs configuring" or "this is already safe". The new entries added in this PR don't.

* Concepts seems perpendicular
* Configure Global Security just discusses the UI
* Other Security Topics is just a random catch-all.
* Exposed services and ports